### PR TITLE
Muon chambers bitmap

### PR DESCRIPTION
**Pull request by nicolovalle**
Bitmap with fired muon chambers added in StoredFwdTracks (Framework/Core/include/Framework/AnalysisDataModel.h) and ReducedMuonExtra table (Analysis/DataModel/include/AnalysisDataModel/ReducedInfoTables.h). PWDQQ table makers also extended.

### CODE REVIEW

File: Framework/Core/include/Framework/AnalysisDataModel.h
Comment (iarsene) [inline Framework/Core/include/Framework/AnalysisDataModel.h]: Since you use it as a bit map, you should make this as unsigned, in this case uint16_t

**Comment by iarsene** [comment]
Hi @nicolovalle, I had one comment on the type used for the bitmap column. Also, please fix the clang and space checker, both fail.

**Review by iarsene — APPROVED** [review]

File: Framework/Core/include/Framework/AnalysisDataModel.h
Comment (jgrosseo) [inline Framework/Core/include/Framework/AnalysisDataModel.h:345]: How many chambers do we have?

Comment (nicolovalle) [reply Framework/Core/include/Framework/AnalysisDataModel.h:345]: Hi, this works with the 10 tracking chambers

**Comment by jgrosseo** [comment]
This changes are good.

Before merging, we have to adjust the RUN3 converter and produce new data. I convert this to draft, so that we do not merge it by accident. Could you take care of the changes?
Add the new field, and fill it accordingly:
https://github.com/alisw/AliPhysics/blob/master/RUN3/AliAnalysisTaskAO2Dconverter.cxx

Once you have the PR in AliPhysics, mention this PR, so that we have the link.

**Comment by nicolovalle** [comment]
Hi, sorry, I forgot to paste the link.
The pull request in Aliphysics is ready: https://github.com/alisw/AliPhysics/pull/17940

**Comment by jgrosseo** [comment]
OK. I approved that one. Then we reproduce the data.

**Review by jgrosseo — APPROVED** [review]

**Comment by jgrosseo** [comment]
@nicolovalle We forgot about this PR. Could you solve the conflict, so that we can merge it?

**Pull request closed without merging**
